hardware.inc

```diff
@@ -810,23 +819,23 @@ def OAMA_X      rb ; 1
     def OAM_X_OFS equ  8 ; subtract 8 from what's written to OAM to get the real X position
 def OAMA_TILEID rb ; 2
 def OAMA_FLAGS  rb ; 3
-    def OAMB_PRI     equ 7 ; whether the OBJ is drawn above BG colors 1-3
-    def OAMB_YFLIP   equ 6 ; whether the whole OBJ is flipped vertically
-    def OAMB_XFLIP   equ 5 ; whether the whole OBJ is flipped horizontally
-    def OAMB_PAL1    equ 4 ; (DMG only) which of the two palettes the OBJ uses
-    def OAMB_BANK1   equ 3 ; (CGB only) which VRAM bank the OBJ takes its tile(s) from
-    def OAMF_PALMASK equ %00000_111 ; (CGB only) which palette the OBJ uses
-        def OAMF_PRI   equ 1 << OAMB_PRI
-        def OAMF_YFLIP equ 1 << OAMB_YFLIP
-        def OAMF_XFLIP equ 1 << OAMB_XFLIP
-        def OAMF_PAL0  equ 0 << OAMB_PAL1
-        def OAMF_PAL1  equ 1 << OAMB_PAL1
-        def OAMF_BANK0 equ 0 << OAMB_BANK1
-        def OAMF_BANK1 equ 1 << OAMB_BANK1
-def OBJ_B     rb 0 ; size of OBJ in bytes = 4
+    def B_OAM_PRIORITY equ 7 ; whether the OBJ is drawn above BG colors 1-3
+    def B_OAM_YFLIP    equ 6 ; whether the whole OBJ is flipped vertically
+    def B_OAM_XFLIP    equ 5 ; whether the whole OBJ is flipped horizontally
+    def B_OAM_PAL1     equ 4 ; (DMG only) which of the two palettes the OBJ uses
+    def B_OAM_BANK1    equ 3 ; (CGB only) which VRAM bank the OBJ takes its tile(s) from
+    def OAM_PALMASK equ %00000_111 ; (CGB only) which palette the OBJ uses
+        def OAM_PRIORITY equ 1 << B_OAM_PRIORITY
+        def OAM_YFLIP    equ 1 << B_OAM_YFLIP
+        def OAM_XFLIP    equ 1 << B_OAM_XFLIP
+        def OAM_PAL0     equ 0 << B_OAM_PAL1
+        def OAM_PAL1     equ 1 << B_OAM_PAL1
+        def OAM_BANK0    equ 0 << B_OAM_BANK1
+        def OAM_BANK1    equ 1 << B_OAM_BANK1
```

Comment:
Maybe it would make sense to give these names that reflect that you can also use these for background tiles? I've been using OAMF_XFLIP and friends when writing to background tile attributes, because these defines are already there, and it looks goofy but you can still understand what it's doing so I don't think that's necessarily a problem.